contracts/tokens/ERC721BridgeToken.sol

```diff
@@ -77,6 +77,29 @@ contract ERC721BridgeToken is ERC721, IBurnableMintableERC721Token {
         _burn(_tokenId);
     }
 
+    // hack to access private fields in ERC721 contract
+    struct MetadataStorage {
+        string name;
+        string symbol;
+    }
+
+    /**
+     * @dev Updated bridged token name/symbol parameters.
+     * Only bridge owner or bridge itself can call this method.
+     * @param _name new name parameter, will be saved as is, without additional suffixes like " from Mainnet".
+     * @param _symbol new symbol parameter.
+     */
+    function setMetadata(string calldata _name, string calldata _symbol) external onlyOwner {
+        require(bytes(_name).length > 0 && bytes(_symbol).length > 0);
+
+        MetadataStorage storage metadata;
+        assembly {
+            metadata.slot := 6
```

Comment:
Is it safe to assume that _name will always be in the slot 6?

Reply:
It must be there, since our proxy storage uses slot 6 for the name, slot 7 for the symbol, as well as the implementation does the same. We cannot change this slot, since we already have deployed tokens which we plan to upgrade over time.

But generally, we need to be very careful when updating our code base and always ensure that storage slots in the code are not changed.

Reply:
Actually, we can rewrite this hack in a normal way using regular assignments after the merge of PR #42. It was necessary to change some portion of openzeppelin's contract there, so the inherited ERC721 contract will be stored directly in the repo. I just don't want to perform the same changes under two PR's.